src/AVFoundation/AVCaptureDeviceDiscoverySession.cs

```diff
@@ -19,7 +21,8 @@ public static AVCaptureDeviceDiscoverySession Create (AVCaptureDeviceType [] dev
 		{
 			var arr = new NSMutableArray ();
 			foreach (var device in deviceTypes)
-				arr.Add (device.GetConstant ());
+				if (device.GetConstant () is NSObject deviceType)
+					arr.Add (deviceType);
```

Comment:
This is also a change in behavior:

```suggestion
				arr.Add (device.GetConstant ()!);
```